LibC/unistd.h

```diff
@@ -109,4 +109,7 @@ enum
  */
 #define _POSIX_PRIORITY_SCHEDULING
 
+// Stifle an less error iirc
+#define _PC_VDISABLE 8
```

Comment:
Remove the comment, and move this to sit in the enum with _PC_NAME_MAX in the same file (since they are used in the same function) {f}pathconf.

Reply:
Gotcha.